library/ecdsa.c

```diff
@@ -400,6 +400,9 @@ int mbedtls_ecdsa_read_signature( mbedtls_ecdsa_context *ctx,
                               &ctx->Q, &r, &s ) ) != 0 )
         goto cleanup;
 
+    /* At this point we know that the buffer starts with a valid signature.
+     * Return 0 if the buffer just contains the signature, and a specific
+     *  error code the valid signature is followed by more data. */
```

Comment:
Typo: ... code IF the ...

Reply:
Amended. I amended the commit: f6c32ad91ec00a48a1023cd5ce33b78e412f1747 → 5114d3e4e1cb6e5a71ceafa56dd7da5f9182f9d9